app/Services/ExportService.php

```diff
@@ -19,6 +19,10 @@ class FreshRSS_Export_Service {
 	/** @var FreshRSS_TagDAO */
 	private $tag_dao;
 
+	const FRSS_NAMESPACE = 'https://freshrss.org/#v1';
```

Comment:
@marienfressinaud Any comment? :-)
Alternatives include http://purl.org , maybe http://xmlns.com, or a non-URL URI

Reply:
https://freshrss.org/opml ? (I will not be able to redirect the `#v1` hash)

Reply:
https://freshrss.org/opml looks good @marienfressinaud :-)
(Of course for `#v1` but it was to avoid introducing a non-working URL so far)

Reply:
Ok, let me know when the documentation page is ready, I'll do the redirection then

Reply:
https://github.com/FreshRSS/FreshRSS/pull/4342/commits/54cb9388c6e60145236f7ba9799b1d3881bab58e

Reply:
@marienfressinaud Here you are https://freshrss.github.io/FreshRSS/en/developers/OPML.html

Reply:
Done! https://freshrss.org/opml (code 302 for now)

Reply:
Super 👍🏻